### PR TITLE
Add a Code of Conduct

### PR DESCRIPTION
**Pull request by akrabat**
See #2324

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Code of Conduct
+
+The Slim Framework code of conduct is derived from the Ruby code of conduct. This document provides community guidelines for a safe, respectful, productive, and collaborative place for any person who is willing to contribute to the Slim Framework community. It applies to all “collaborative space”, which is defined as community communications channels (such as mailing lists, IRC, Slack, forums, submitted patches, commit comments, etc.). Any violations of the code of conduct may be reported by contacting one or more of the project maintainers.
```
Comment (geggleto) [inline CODE_OF_CONDUCT.md]: I think we should probably be a bit more specific about how this is possible.

What about adding all of us to the Authors list in composer.json?

Comment (akrabat) [reply CODE_OF_CONDUCT.md]: Let's get a generic email address set up.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+* Participants will be tolerant of opposing views.
+* Participants must ensure that their language and actions are free of personal attacks and disparaging personal remarks.
+* When interpreting the words and actions of others, participants should always assume good intentions.
+* Behaviour which can be reasonably considered harassment will not be tolerated.
```
Comment (codeguy) [inline CODE_OF_CONDUCT.md]: What does "reasonably" mean? That's really the only thing I see that will likely prompt push back. Otherwise this looks great.

Comment (geggleto) [reply CODE_OF_CONDUCT.md]: that's a great question; I think reasonably will end up being what we (maintainers) believe is the correct thing under the context...

Minimally, I think what this document is saying is: `Be Nice, Courteous and Understanding`

**Review by geggleto — APPROVED** [review]

**Review by silentworks — APPROVED** [review]

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (silentworks) [reply CODE_OF_CONDUCT.md]: I think what @geggleto suggested is probably better to use as `reasonably` is subjective.

**Comment by akrabat** [comment]
Updated wording of last point. Is that better @silentworks, @geggleto, @codeguy?

**Comment by geggleto** [comment]
👍 LGTM @akrabat

**Pull request merged**